packages/editor/src/components/provider/index.js

```diff
@@ -334,7 +344,9 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 								<>
 									<PatternsMenuItems />
 									<TemplatePartMenuItems />
-									<ContentOnlySettingsMenu />
+									{ ! isZoomOut && (
+										<ContentOnlySettingsMenu />
+									) }
```

Comment:
Would it be better to subscribe to state only in the `ContentOnlySettingsMenu`? That would isolate the subscription and also keep it closer to the affected UI.

The downside would be that any subscriptions in `ContentOnlySettingsMenu` would be run.